data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/RequestMapper.java

```diff
@@ -107,6 +112,19 @@ public RequestMapper(
 
   @Override
   public void handle(final HttpServerRequest request) {
+    // This implements the knative/networking data-plane contract.
```

Comment:
Maybe you can implement this as decorator like the `SimpleProbeHandlerDecorator`?

Reply:
Sure.